keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/persistence/KeelReadOnlyRepository.kt

```diff
@@ -49,6 +49,8 @@ interface KeelReadOnlyRepository {
 
   fun isRegistered(name: String, type: ArtifactType): Boolean
 
+  fun artifactVersions(artifact: DeliveryArtifact, limit: Int): List<String>
```

Comment:
Why not add the limit to the existing function below, with a default value?

Reply:
tried it and it asked me to change all the other places where this function is being called... did I miss something? 😅

Reply:
Did you assign a default value for the parameter?

Reply:
Ya, you'll need to assign a default value to the parameter ( `..., limit: Int = 30)`)

Reply:
🤦